lib/setErrorHeaders.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const { kReplyErrorHandlerCalled } = require('./symbols.js')
+
+function setErrorHeaders (error, reply, res) {
```

Comment:
Can you please move the extraction of `res` from `reply` inside the function?

```suggestion
function setErrorHeaders (error, reply) {
  const res = reply.raw
```